src/planscape/conditions/management/commands/load_rasters.py

```diff
@@ -0,0 +1,45 @@
+from django.core.management.base import BaseCommand, CommandParser
+from conditions.models import Condition
+from conditions.registry import register_condition_raster
+
+
+class Command(BaseCommand):
+    help = "Loads conditions from 'conditions.json' into the database."
+
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "--condition-ids",
+            nargs="+",
+            type=int,
+        )
+
+        parser.add_argument(
+            "--clear",
+            default=True,
+            action="store_true",
+            help="If set, it will remove the previous raster data from the raster table before loading the new ones."
```

Comment:
Might be more helpful if you phrase it as "If set to false, will not remove the previous raster data from the raster table before loading the new ones" since the default is True